nodejs/v3/nestjs-prisma/commands/run

```diff
@@ -20,14 +17,17 @@ mono build
 
 cd /app
 
-echo "Installing app dependencies"
-npm install
-
 echo "Linking integration"
 npm link @appsignal/nodejs
 
+echo "Installing app dependencies"
+npm install
+
 echo "Building app"
```

Comment:
I would expect this change in order to overwrite the locally linked `@appsignal/nodejs` package. That's the behavior I've seen before. Can we revert this command order and does it work then?

Reply:
If I recall correctly, `npm link foo` means "link that package _and install everything else in the dependencies_", that is, it's also `npm install`. `npm link` also saves that this dependency is linked, so running `npm install` afterwards respects the link.

So this `npm install` following it, I think, is redundant.

If we do it the other way around, it works, but it downloads the dependencies for the `@appsignal/nodejs` version uploaded to the npm servers, then re-installs with the version in the integration folder.

I really want to re-do these scripts for all the test setups, but I also don't want to open that can of worms right now. We do way too many `npm install` calls. We should only need the one `npm link` call.